ElementX/Sources/Screens/HomeScreen/View/HomeScreenRoomCell.swift

```diff
@@ -20,6 +20,8 @@ struct HomeScreenRoomCell: View {
     let room: HomeScreenRoom
     let context: HomeScreenViewModel.Context
     
+    @Environment(\.dynamicTypeSize) var size
```

Comment:
One last suggestion looking through this again. Its nice to keep the name of environment values consistent with their keys:
```suggestion
    @Environment(\.dynamicTypeSize) private var dynamicTypeSize
```